f5router/bigipResources/bigipResources.go

```diff
@@ -131,6 +133,12 @@ type (
 		Strategy    string   `json:"strategy"`
 	}
 
+	// IRule definition
```

Comment:
Verify this matches k8s ctlr

Reply:
k8s-bigip-ctlr has a partition field in addition to the fields we have here. This should be fine for now since the cf-bigip-ctlr can only handle one partition.

Reply:
It's fine as we create all our objects under the partition map, we don't want partition in any object, k8s is changing that.

Reply:
cool thanks Dan.